spec/plans/report.fmf

```diff
@@ -76,13 +76,17 @@ description:
         tmt run --all report --how polarion --project-id TMT --no-upload --file test.xml
 
         # Use polarion as the default report for given plan
+        # Make sure the values of custom fields can be accepted by Polarion
         report:
             how: polarion
             file: test.xml
             project-id: TMT
             title: tests_that_pass
-            planned-in: RHEL-9.1.0
+            # Use RHEL-9_4_0 instead of RHEL-9.4.0 which is not accepted by Polarion
+            planned-in: RHEL-9_4_0
             pool-team: sst_tmt
+            # Use x8664 instead of x86_64
+            arch: x8664
```

Comment:
Is this particular architecture and distro naming tied to a specific Polarion instance or is the limitation (e.g. not possible to use dots in the name) hard coded in the Polarion implementation?

Reply:
@KwisatzHaderach, do you possibly know any details about this?

Reply:
our project and likely RHEL as well have the arch without '_' but it's not because polarion wouldn't support 'x86_64', just tried to change the id of that arch in administration and it worked fine, same for adding '.'
for planned-in our project also uses '_' instead of '.', but this is one is special as it's collected from other fields and not immediately visible what values it accepts, will differ for each project

Reply:
Would it make sense to use wording that suggests this might be needed, depending on your project setup? Because written like this it makes an impression this way and this way only is accepted by each and every Polarion instance and project. Maybe a dedicated example, "your project might need custom `arch` and `planned-in`".

Reply:
good idea, only not just for those 2 fields, same applies to virtually any other field